dev-requirements.txt

```diff
@@ -5,6 +5,7 @@ dbt-tests-adapter~=1.7.4
 flake8~=6.1
 Flake8-pyproject~=1.2
 isort~=5.13
+mmh3~=4.0.1
```

Comment:
```suggestion
```
I believe adding it to `setup.py` is enough

Reply:
I hope so, thanks 🤝